crates/bitvm2/src/bin/mock.rs

```diff
@@ -0,0 +1,180 @@
+use bitcoin::{consensus::encode::serialize_hex, Amount, Network, OutPoint, PrivateKey, PublicKey, Transaction, Txid};
+use bitvm::chunk::api::NUM_TAPS;
+use bitvm2_lib::{operator, committee, 
+    types::{CustomInputs, Bitvm2Parameters},
+};
+use bitvm::treepp::*;
+use musig2::{AggNonce, PartialSignature, PubNonce, SecNonce};
+use secp256k1::SECP256K1;
+use std::str::FromStr;
+use goat::{contexts::base::generate_n_of_n_public_key, scripts::generate_burn_script_address};
+use goat::transactions::base::Input;
+use goat::transactions::pre_signed::PreSignedTransaction;
+
+pub fn main() {
```

Comment:
You can consider this either integration test, or an example, but now a bin in a library

Reply:
bin removed, more test cases will be added later